components/automate-cli/cmd/chef-automate/upgrade.go

```diff
@@ -170,8 +155,26 @@ func runAutomateHAFlow(args []string, offlineMode bool) error {
 				return err
 			}
```

Comment:
check the closing braces

Reply:
seems correct to me and it is compiling also